rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -168,13 +167,15 @@ public Set<String> supportedCapabilities() {
 [[cluster_features]]
 === Require or skip cluster features
 
-In the past, Elasticsearch has been over-reliant on release version checks all across its code base.
-This has become a limiting factor when deploying Elasticsearch in a more continuous fashion.
+Cluster features indicate a particular top-level functionality and are used for _internal_ checks and coordination
+within Elasticsearch to enable functionality once available on all nodes, e.g. usage of a new transport endpoint.
 
-Moving forward, the goal is to base any such check on the availability of a cluster feature.
-Rather than <<skip_version, skipping tests based on various release version ranges>>, you can
-require `cluster_features` to be either present (`requires`) and / or absent (`skip`) in order to run a test.
-For instance:
+Other than <<capabilities_check, capabilities>>, cluster features are strictly internal, though can also be used for
```

Comment:
I'm not familiar with "Other than" used to express "Differently from", but I'll let a native English speaker decide here :)

Reply:
'Other than' implies that the other thing is a subset of the main group you're talking about. The text as written implies that capabilities are a subset or a type of of cluster features.

I suggest 'In contrast to', which highlights the difference without imposing any other relation between the two concepts.